### PR TITLE
manifest: remove links from services

### PR DESCRIPTION
**Pull request by ddollar**

### CODE REVIEW

**Comment by ddollar** [comment]
[remove links from services](https://trello.com/c/1bvqcAjN/498-remove-links-from-services)

File: docs/guides/service-discovery.md [hunk 1]
```diff
@@ -36,30 +36,28 @@ For an app named `myapp` with a `convox.yml` like this:
...
+> If you want this connection to be encrypted you would need to handle SSL inside the Service.
 
 DNS search suffixes are automatically configured for internal hostnames on a Rack. The following URLs would
 also work for contacting the `auth` Service:
 
 * `http://auth:5000` for Services on the same app.
 * `http://auth.convox-myapp:5000` for other apps on the same Rack.
 
-> The `convox` portion of the internal hostnames in the examples above is the name of the Rack.
+> The `convox` portion of these internal hostnames is the name of the Rack.
```
Comment (beastawakens) [inline docs/guides/service-discovery.md]: Is it worth specifying this like:
http://{SERVICE NAME}.{RACK NAME}-{APP NAME}.svc.cluster.local:{PORT}
to make it clear?

Comment (beastawakens) [reply docs/guides/service-discovery.md]: Well, and `http://{SERVICE NAME}:{PORT}` as well 😄

Comment (ddollar) [reply docs/guides/service-discovery.md]: I think it's more clear to have the specific examples.

**Pull request merged**